### DISCUSSION TITLE
wel package error in loading an exsiting model

### DISCUSSION THREAD

**Discussion by zargol7419** [modflowpy/flopy#1532] [Q&A] [answered]
Hi,
I am using flopy package to run an existing model which I have change the content of well package in but when it comes to running well package in my code I got an error but I can't  find the reason. Could you please help me in this regard?
Thanks in advance for your answers.
Here is my simple code:

[import flopy
import os
import sys
first = flopy.modflow.Modflow.load("E:\\zargolooo\\test-code\\modflow-test\\step6_MODFLOW\\Out_Mf2k 3\\step6.mfn", version='mf2k',exe_name="D:\\GMS\\models\\mf2k\\usgs\\mf2k.exe", verbose =True , model_ws = "E:\\zargolooo\\test-code\\MODFLOW\\step6_MODFLOW\\step6._ws" , load_only=None , forgive=False , check =False )](url)

And here is the error:
![image](https://user-images.githubusercontent.com/103249900/188556126-dd897789-03f9-435f-be66-b899430f5b6c.png)

![image](https://user-images.githubusercontent.com/103249900/188556203-58b35cd7-bb09-41d4-a273-e5394ebf77f2.png)

**Comment by wpbonelli** [comment, marked as answer]
@zargol7419 it looks like you are using a Python 2 environment. Python 3 is required to use the latest version of FloPy. Does the error still occur with Python 3?

**Reply by zargol7419** [reply]
Actually I have installed python3 as you told me before (when I asked another question about GBOB package) but the name of anaconda's folder is still python2. I have checked the version of my python several times so I guess it should be originated from another problem.

**Reply by jdhughes-dev** [reply]
GMS MODFLOW files may not be standard MODFLOW files. I would look at the `wel` file to make sure it matches the [standard well package format](https://water.usgs.gov/nrp/gwsoftware/modflow2000/Guide/wel.html).
